model/config_repo.py

```diff
@@ -23,6 +23,13 @@ class KubernetesSecretTarget:
     secret_key: str
 
 
+@dataclasses.dataclass
+class SugarSolinasServiceTarget:
+    type: str
+    team_id: str
+    github: str
+
+
 @dataclasses.dataclass
 class ReplicationMapping:
     target: SecretsServerTarget | KubernetesSecretTarget
```

Comment:
Also reference `SugarSolinasServiceTarget` for sake of deserialisation.

Reply:
fixed